app/controllers/measurements_controller.rb

```diff
@@ -4,11 +4,14 @@ class MeasurementsController < ApplicationController
 
   # GET /measurements
   # GET /measurements.json
+  # rubocop:disable Metrics/AbcSize
   def index
     @measurements = Measurement.all
     @measurements = @measurements.where(sensor_id: params[:sensor_id].split(',')) if params[:sensor_id].present?
+    @measurements = @measurements.last_per_sensor(params[:last_per_sensor]) if params[:last_per_sensor].present?
     @measurements = @measurements.last(params[:last]) if params[:last].present?
```

Comment:
It returns the last *n* results which are the newest ones. `limit` would be like `first(n)`.